pages/proposals/form/fields.js

```diff
@@ -0,0 +1,201 @@
+import validator from './validator';
+
+// *** IMPORTANT ***
+//
+// make sure all keys are all lowercase and contain no symbols or spaces
+// reference: https://www.npmjs.com/package/google-spreadsheet#row-based-api-limitations
+//
+// **********************************************
+
+let partOneFields = {
+  'firstname': {
+    type: `text`,
+    label: `First name`,
+    placeholder: `First name`,
+    labelClassname: `required`,
+    fieldClassname: `form-control`,
+    validator: [
+      validator.emptyValueValidator()
+    ]
+  },
+  'surname': {
+    type: `text`,
+    label: `Surname`,
+    placeholder: `Surname`,
+    labelClassname: `required`,
+    fieldClassname: `form-control`,
+    validator: [
+      validator.emptyValueValidator()
+    ]
+  },
+  'email': {
+    type: `text`,
+    label: `Email address`,
+    placeholder: `hello@example.com`,
+    labelClassname: `required`,
+    fieldClassname: `form-control`,
+    validator: [
+      validator.emptyValueValidator(),
+      validator.emailValidator()
+    ]
+  },
+  'organisation': {
+    type: `text`,
+    label: `Organisation `,
+    placeholder: `Organisation `,
+    fieldClassname: `form-control`,
+    validator: []
+  },
+  'twitterhandle': {
```

Comment:
As a user, I'm going to be confused about whether I need to add the @ symbol or not